app/assets/javascripts/shared/datatable/state.js

```diff
@@ -1,59 +1,84 @@
 $(document).on('preInit.dt', 'body.issues.index', function (e, settings) {
   var $paths = $('[data-behavior~=datatable-paths]');
-  if ($paths.data('table-state-url') == undefined) { return; }
+  if ($paths.data('table-state-url') == undefined) {
+    return;
+  }
+
+  var api = new $.fn.dataTable.Api(settings),
+    isQA = $('[data-behavior~=qa-viewer]').length > 0;
```

Comment:
Can we define the `isQA` variable in the `setupStateButtons` method instead of passing it?